extract_endpoint/src/extract_endpoint/post_to_endpoint.py

```diff
@@ -31,10 +36,32 @@ def post_stream(stream: typing.IO[bytes],
                          data={'salt': salt, 'signature': signature, 'timestamp': timestamp})
 
 
+def send_to_tl_app(url: str, data: typing.Dict[str, str]) -> int:
+    return requests.post(url, data=data).status_code
+
+
+def trigger_tl(url) -> int:
+    salt = secrets.token_hex(16)
+    hasher = hashlib.new('sha3_256')
+    hasher.update((salt + _etl_secret_key()).encode())
+    signature = hasher.hexdigest()
+    data = dict(salt=salt, signature=signature)
+    return send_to_tl_app(url, data)
```

Comment:
I don't really see the point in wrapping just a single POST in a function call, any reasoning behind that?

Reply:
Also, it seems odd to have this much functionality in the the CLI file. Should this stuff be refactored into it's own file, and the CLI left to just do CLI stuff?

Reply:
I figured they were below the threshold for doing that, ie they're just doing a hash and a post.